packages/discord.js/src/managers/MessageManager.js

```diff
@@ -49,6 +49,8 @@ class MessageManager extends CachedManager {
 
   /**
    * Options used to fetch multiple messages.
+   * <info>The `before`, `after`, and `around` parameters are mutually exclusive,
+   * only one may be passed at a time.</info>
```

Comment:
Also (mutually exclusive)[https://www.collinsdictionary.com/dictionary/english/mutually-exclusive] is self explaining,
so no need for this line

Reply:
I used the same text they have on Discord Documentation: https://discord.com/developers/docs/resources/channel#get-channel-messages

Reply:
Like below djs only mentions that it is mutually exclusive
https://github.com/discordjs/discord.js/blob/1b29099ed0b0deb98db844671aa23b4a84ec9c08/packages/discord.js/src/managers/ThreadManager.js#L78

Reply:
updated it